server/server/models/user.py

```diff
@@ -104,7 +104,11 @@ def model_to_json(sqlalchemy_object):
                   if isinstance(prop, ColumnProperty)]
     _dict = {}
     for key in fields_arr:
-        _dict[key] = str(getattr(sqlalchemy_object, key))
+        temp = getattr(sqlalchemy_object, key)
+        if isinstance(temp, datetime.datetime):
+            _dict[key] = str(getattr(sqlalchemy_object, key))
```

Comment:
You can use your `temp` variable here, it will be better than calling `getattr` method again. Also please change the `temp` name on something more informative.